azext_iot/central/_help.py

```diff
@@ -85,6 +85,37 @@ def _load_central_devices_help():
             --device-id {deviceid}
     """
 
+    helps[
+        "iot central device manual-failover"
+    ] = """
+        type: command
+        short-summary: Execute a manual failover of device across multiple IoT Hubs to validate device firmware's ability to reconnect using DPS to a different IoT Hub.
+        long-summary: For more information about high availability visit https://github.com/iot-for-all/iot-central-high-availability-clients#readme
```

Comment:
While I do love this doc (its hilarious), should we perhaps considering linking to an official azure docs page instead?

This is the official azure CLI, we shouldn't be linking to one off repos

Reply:
it's official one which i got from team